api/clans.py

```diff
@@ -0,0 +1,24 @@
+import faf.db as db
+
+from api import *
+
+@app.route('/clans')
+def clans_get():
+    clans = []
```

Comment:
This declaration is superfluous, please pay attention to the warnings in the IDE